nextjs-task-system/app/user/page.tsx

```
@@ -0,0 +1,345 @@
+          "In Progress": 0,
+          Completed: 0,
+        },
+        byPriority: {
+          Low: 0,
+          Medium: 0,
+          High: 0,
+        },
+      };
+
+      tasks.forEach((task: Task) => {
+        if (summary.byStatus.hasOwnProperty(task.status)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)